source/imaer-shared/src/main/java/nl/overheid/aerius/shared/domain/calculation/NCACalculationOptions.java

```diff
@@ -27,23 +27,13 @@ public class NCACalculationOptions implements Serializable {
 
   private static final long serialVersionUID = 1L;
 
-  private ProjectCategory projectCategory;
+  private String projectCategory;
 
   /**
    * Regional area for which a permit calculation would apply to.
    */
   private String permitArea;
 
-  /**
-   * Location of the meteorological site location.
-   */
-  private String meteoSiteLocation;
-
-  /**
-   * List of meteo years to calculate.
-   */
-  private List<String> meteoYears = new ArrayList<>();
-
```

Comment:
Where will these be saved, if not here?

Reply:
These are already in ADMSOptions. Looks like this is just some cleanup.

Reply:
Correct, the fields were not used (there were no getter/setters anymore) so should be save to remove.